<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>598</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>598</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>576</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/newPrefix/Images/ShowDisorders.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_records_pathology">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>415</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
background-color: rgb(157, 180, 255);
border-radius: 20px;

color:rgb(69, 77, 179);
font-size: 14px;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Pathology Records</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/patient.svg</normaloff>Icons/patient.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_records_session">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>285</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
background-color: rgb(157, 180, 255);
border-radius: 20px;

color:rgb(69, 77, 179);
font-size: 14px;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Session Records</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/session.svg</normaloff>Icons/session.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_main">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>220</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
background-color: rgb(157, 180, 255);
border-radius: 20px;

color:rgb(69, 77, 179);
font-size: 14px;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Dashboard</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/dashboard.svg</normaloff>Icons/dashboard.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>505</x>
      <y>91</y>
      <width>441</width>
      <height>401</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Age</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Gender</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Phone</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Email</string>
     </property>
    </column>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>180</y>
      <width>185</width>
      <height>345</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(157, 180, 255);
border-radius: 20px;
</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>168</width>
       <height>229</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>149</width>
       <height>229</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url();
background-color: rgb(255, 252, 255);
border-radius: 10px;
text-align:center;

color:rgb(69, 77, 179);
font-size: 14px;
font-weight: bold;

Height:40px</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_records_patient">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>500</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
background-color: rgb(157, 180, 255);
border-radius: 20px;

color:rgb(69, 77, 179);
font-size: 14px;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Patient Records</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/patient.svg</normaloff>Icons/patient.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
